src/widgets/block.rs

```diff
@@ -311,8 +377,9 @@ impl<'a> Widget for Block<'a> {
                 .set_style(self.border_style);
         }
 
-        // Title
```

Comment:
This method is getting long - can we extract the border rendering and the title rendering into two methods?

Reply:
For now, I just separated them in two functions: `render_border` and `render_titles` in the block implementation. Later, I look for a better way of handling this.